src/include/detail/ivf/index.h

```diff
@@ -56,6 +57,7 @@ int ivf_index(
     tiledb::Context& ctx,
     const ColMajorMatrix<T>& db,
     const std::vector<ids_type>& external_ids,
+    const std::vector<ids_type>& deleted_ids,
```

Comment:
I am not quite sure I understand this?  Shouldn't there be an updates array that includes additions and deletions, plus a validity array?  This was how it was described in https://www.notion.so/TileDB-Vector-search-support-for-updates-9ef808bcc5c24e7197403bc19bc342d1?pvs=4 (iiuc).

The interface I created for queries now takes an updates array, update ids, and a validity array.

Reply:
Yes this is implemented on the Python side for now. The cpp ingestion just takes a list of `deleted_ids` to ignore from the input. This is mainly used during consolidation of updates and previous data. The previous data need to be filtered with this set of ids.